src/cli/cli.rs

```diff
@@ -10,9 +10,11 @@ use clap::{
     builder::PossibleValuesParser,
     error::Error as ClapError,
     error::ErrorKind,
-    {ArgMatches, Args, Command, FromArgMatches},
+    {ArgMatches, Args, Command, FromArgMatches, Subcommand},
 };
+use clap_complete::dynamic::shells::CompleteCommand;
 use log::debug;
+use std::process;
```

Comment:
nit: Please use the same include order as the rest of the project:

```
std includes
<newline>
ext crates includes
<newline>
this crate includes
```

Reply:
I'm not sure if this is a convention for `rust`. If not, maybe we can add this to `Coding style` part of CONTRIBUTING.md

Reply:
Rust defines includes can be grouped but I don't think it defines the exact above order; the above is not uncommon though. Feel free to add a word about this in the contributing guide.